spec.html

```diff
@@ -39843,6 +39857,82 @@ <h1>Array.from ( _items_ [ , _mapper_ [ , _thisArg_ ] ] )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-array.fromAsync">
```

Comment:
esids should be lowercase

```suggestion
      <emu-clause id="sec-array.fromasync">
```